ssl/ssl_ciph.c

```diff
@@ -2216,8 +2226,13 @@ int ssl_cipher_get_overhead(const SSL_CIPHER *c, size_t *mac_overhead,
 
 int ssl_cert_is_disabled(SSL_CTX *ctx, size_t idx)
 {
-    const SSL_CERT_LOOKUP *cl = ssl_cert_lookup_by_idx(idx);
+    SSL_CERT_LOOKUP *cl;
+
+    /* A new provider-loaded key type is always enabled */
```

Comment:
Drop the word "new" here. It won't be "new" any more very quickly once this is merged.

Reply:
So changed in 0ac814f8247d1a5f157eec2b62bce0000c3a2d1e.